generate_openj9_scc.sh

```diff
@@ -0,0 +1,295 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+unset OPENJ9_JAVA_OPTIONS
+
+# Default size for SCC
+SCC_SIZE="50m"
+
+# Runs for generating SCC
+SCC_GEN_RUNS_COUNT=3
+
+# is running on ubuntu or debian - package manager apt
+IS_APT_ENV=false
+# Intitialise default / allowed sample app status
+RUN_ECLIPSE=false
+RUN_TOMCAT=true
+
+# Intitialise default / required packages status
+INSTALL_GTK=false
+INSTALL_XVFB=false
+
+# App download locations
+DOWNLOAD_PATH_ECLIPSE=/tmp/eclipse
+DOWNLOAD_PATH_TOMCAT=/tmp/tomcat
+
+# App installation locations
+INSTALL_PATH_ECLIPSE="$HOME"/eclipse-home
+INSTALL_PATH_TOMCAT="$HOME"/tomcat-home
+
+# function to install packages
+function install_packages() {
+    if [ "$IS_APT_ENV" == true ]; then
```

Comment:
Given that, we need to add support for every package mgr in the future, it would be good to have `apt` specific code in a separate function. Similarly for other package managers. `install_packages` would then call them all.

Reply:
created a `apt` specific function